jenkins/Jenkinsfile.nightly

```diff
@@ -13,60 +13,71 @@ pipeline {
         stash name: "build", includes: "build/**";
       }
     }
-    stage("Nightly Tests") {
-      parallel {
-        stage("Detailed Router ISPD") {
-          agent any;
-          stages {
-            stage("Clean-up ISPD") {
-              steps {
-                sh "rm -rf /home/jenkins/ispd/runs/*";
-              }
-            }
-            stage("Test ISPD") {
-              steps {
-                unstash 'build';
-                sh "./src/drt/test/run-ispd.tcl";
-              }
-            }
+
+    stage('Test Matrix') {
+      matrix {
+        agent any;
+        axes {
+          axis {
+            name 'ISPD_YEAR';
+            values '18', '19';
+          }
+          axis {
+            name 'TEST_TO_RUN';
+            values '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', 'GPL';
```

Comment:
GPL has nothing to do with ispd and should be kept in a separate stage as it was before (it is just big placement tests).

Reply:
Yes, that is correct. This is a workaround to make all the tests run in parallel. The two other alternatives are to run first the ISPD and then GPL or refactor the file to be verbose (i.e., write each of the 10 tests in a stage).

Reply:
can't you wrap ispd & gpl in a parallel?

Reply:
No, looks like Jenkins matrix does not allow to be inside a parallel block.
https://www.jenkins.io/doc/book/pipeline/syntax/#declarative-matrix